src/Microsoft.AspNetCore.Razor.Evolution/Intermediate/HtmlContentIRNode.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Evolution.Legacy;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.Intermediate
+{
+    public sealed class HtmlContentIRNode : RazorIRNode
+    {
+        public string Content { get; set; }
+
+        public override IList<RazorIRNode> Children { get; } = new List<RazorIRNode>();
```

Comment:
Will want to comb through these IR Nodes and not always allocate a list for the children. In the 99% case this node will have 0 children.